README.md

```diff
@@ -1,2 +1,137 @@
+
+## Requirements
+Requires Starcounter 2.4.0.6535 or later and .NET Framework 4.6.1.
+
+## How to use
+First, in your Starcounter app project, install Starcounter.Uniform trough NuGet: `Install-Package Starcounter.Uniform` and add a reference to `Starcounter.Uniform.dll`.
+
+This library provides helping method for a variety of uniform components and way to use it is slightly different for each one of them.
+
+## Uni-data-table
+`uni-data-table` is a custom element that is an adapter to `vaadin-grid`. It provides rich view of a data table which contain funcionalities like:
+- Pagination or infinite scrolling.
+- Sorting.
+- Support for variety of column types.
+
+More about `uni-data-table` component you can read in Uniform.css [readme for uni-data-table](https://github.com/Starcounter/uniform.css/tree/master/components/uni-data-table).
+
+### Basic usage
+To cover most of the cases we provide default implementation of all needed components to generate data table structure. Although `uni-data-table` implementation is fully customizable and user can reimplement all of its part to work as he want it to.
+
+To create basic `uni-data-table` structure:
+
+1. Provide container for data table structure in your json file:
+
+```json
+{
+  "DataTable": {}
+}
+
+```
+
+2. Create your data table row view-model, e.g.:
+```json
+{
+  "FirstName": "",
+  "LastName": "",
+  "Email$": ""
+}
+```
+
+3. In your handler where you return your view-model that contains data table, add data table inicialization with `DataTableBuilder`.
+```cs
+Handle.GET("/YourAppName/partial/datatable", () =>
+            {
+                return Db.Scope(() =>
+                {
+                    var dataTablePage = new DataTablePage();
+                    dataTablePage.DataTable = new DataTableBuilder<DataTableRowViewModel>()
+                        .WithDataSource(DbLinq.Objects<DataTableRowDataModel>())
+                        .WithColumns(columns =>
+                            columns
+                                .AddColumn(b => b.FirstName,
+                                    column => column.DisplayName("First Name").Sortable().Filterable())
+                                .AddColumn(b => b.LastName, column => column.Sortable().DisplayName("Last Name"))
+                                .AddColumn(b => b.Email,
+                                    column => column.Filterable().Sortable())))
+                        .Build();
+
+                    return dataTablePage;
+                });
+            });
+```
+
+After those steps your container object in view view-model should be populated with our U`niFormItem` view-model that `uni-data-table` component will understand and work with.
+You can find view-model structure used by `uni-data-table` on It's [readme page](https://github.com/Starcounter/uniform.css/tree/master/components/uni-data-table).
```

Comment:
It's => its

Reply:
I would also suggest linking directly to the actual readme: https://github.com/Starcounter/uniform.css/blob/master/components/uni-data-table/README.md